### PR TITLE
Check eps < CUDNN_BN_MIN_EPSILON in FixedBatchNormalization.

### PR DESCRIPTION
**Pull request by cycentum**
I added lines checking the eps value in FixedBatchNormalization as is already done in BatchNormalization. I also added the test case based on the one for functions.batch_normalization.

### CODE REVIEW

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failure is irrelevant to the PR.
